habs_portal/static/css/stations.css

```diff
@@ -10,6 +10,11 @@
   height: 100%;
 }
 
+.plotting-row {
+  height: 300px;
```

Comment:
There's a lot of real-estate we're not utilizing, as seen by the big grey L. I do need to reevaluate the height, you have to specify an actual height value in this element for the child, and plot elements to actually size correctly. Also, we need it because the SVGPlotView uses jQuery after the DOM is loaded to determine the element size.